### PR TITLE
Refactor activities configuration loading

### PR DESCRIPTION
**Pull request by vladimyr**
This PR alters tailor configuration loading by:
- deprecating `.activities-rc.js` & `.activities-rc.json` in favor of `.tailorrc.js` & `.tailorrc.json` configuration filenames
- implementing cli flag (`--config`) & environment variable (`TAILOR_CONFIG`) overrides

Configuration is loaded using [`joycon`](https://npm.im/joycon) under the hood and following filenames are supported (sorted by priority):
- `tailor.config.js`
- `.tailorrc.js`
- `.tailorrc`
- `.tailorrc.json`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,9 +47,28 @@ Use the `.brandrc.example` file as a template: `cp .brandrc.example .brandrc` an
 * `npm run start`
 
 ## :books: Content repository structure
-Repository stucture can be altered through `.activities-rc.json` file. Use the `.activities-rc.json.example` file as a template: `cp .activities-rc.json.example .activities-rc.json` and enter configuration details. It's also possible to use `.activities-rc.js`, by using `module.exports` to export the structure.
-By default, the file is searched for in the root of the project. If a custom location or a custom name is needed, it can be provided through the `activitiesConfig` param to any of the build scripts (without the extension), for example:
-`npm run dev:client -- --activitiesConfig=server/.custom-activities-rc`
+Repository stucture can be altered using activities configuration file.
+Recongnized configuration filenames are (sorted by priority): 
+- `activities.config.js`
```
Comment (underscope) [inline README.md]: Should we rename this to schemas or something along those lines / better ? 🤔

File: README.md [hunk 2]
```diff
@@ -47,9 +47,28 @@ Use the `.brandrc.example` file as a template: `cp .brandrc.example .brandrc` an
...
+and enter configuration details. It is also possible to use a commonjs module, 
+by using `module.exports` to export the repository structure.
+Configuration file is searched for inside current working directory by default. 
+If a custom location or a custom name is needed, it can be provided through the `--activities-config` flag passed to target npm script:
+```
+$ npm run dev:client -- --activities-config=path/to/custom/activities/config.js
+```
+alternatively `TAILOR_ACTIVITIES_CONFIG` environment variable can be used:
+```
+$ TAILOR_ACTIVITIES_CONFIG=path/to/custom/activities/config.js npm run dev:server
```
Comment (underscope) [inline README.md]: Update depending on prev comment change. Consider also removing the prefix.

File: README.md (hunk 1 above)
Comment (vladimyr) [reply README.md]: With backward compatibility support through next few versions? Meaning we warn users they should rename it from `.activities-rc` to `.schemasrc` or `schemas.config.js`?

File: README.md (hunk 2 above)
Comment (vladimyr) [reply README.md]: Removing? 🤔 Shouldn't it become `TAILOR_SCHEMAS_CONFIG`?

**Comment by underscope** [comment]
@vladimyr Please merge latest develop when you get a chance

**Comment by vladimyr** [comment]
> @vladimyr Please merge latest develop when you get a chance

Done ✅

**Pull request merged**